nvi-commons/src/main/java/no/sikt/nva/nvi/common/service/CandidateBO.java

```diff
@@ -436,6 +443,14 @@ private static String mapToUsernameString(Username assignee) {
         return assignee != null ? assignee.value() : null;
     }
 
+    private BigDecimal calculateUndistributedPoints() {
+        return totalPoints.subtract(sumDistributedPoints()).setScale(POINTS_SCALE, ROUNDING_MODE);
+    }
+
+    private BigDecimal sumDistributedPoints() {
+        return institutionPoints.values().stream().reduce(BigDecimal.ZERO, BigDecimal::add);
+    }
+
```

Comment:
I think fronted need both points and undistributedPoints. Maybe rename points to distrubtedPoints